plugins/Kaleidoscope-Colormap-Overlay/src/kaleidoscope/plugin/Colormap-Overlay.h

```diff
@@ -127,8 +131,10 @@ class ColormapOverlay : public kaleidoscope::Plugin {
   }
 
  private:
-  Overlay *overlays_;
-  uint8_t overlay_count_;
+  static uint16_t overlays_base_;
+  static const uint8_t max_overlays_ = 64;  // TODO(EvyBongers): figure this out. How determine a good maximum?
```

Comment:
Following up on https://github.com/keyboardio/Kaleidoscope/pull/1434#discussion_r1991570807:

`layer_count` is far too few, as an overlay only applies to a single key. We could set it to `layer_count * Runtime.device().numKeys()`, but if anyone is gonna need that number of overlays, they're better off using Colormap rather than ColormapOverlay.

Reply:
Understood. I think that starting at a fixed # like 64 seems fine. if we find folks needing it increased, we can do that.